### PR TITLE
Update styles.css

### PR DESCRIPTION
**Pull request by gtkpr**
Fixed:
Issue #13
Footer Scroll Issue

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,4 +1,7 @@
 /* Friends of Girl Develop It Columbus styles */
+:root { /                              /* new */
```
Comment (csteinert) [inline styles.css]: There's an errant "/" after the opening curly-brace, please remove that.

**Review by csteinert — COMMENTED** [review]
Great use of CSS custom properties here! 

Please remove all instances of `/* new */` in this file. I appreciate the intent, but GitHub shows me in the Pull Request view which lines have been changed so they are unnecessary. Also, the comments will no longer be accurate when the next person adds to the file.

**Comment by gtkpr** [comment]
will do!! sorry about that. completely new to github.

File: styles.css (hunk 1 above)
Comment (gtkpr) [reply styles.css]: k that's fixed

**Review by csteinert — APPROVED** [review]
Much better, thanks!

**Pull request merged**